base/process.py

```diff
@@ -591,6 +592,34 @@ def match(node, genotype):
                     partial_matches = filter(lambda x:match(x,[allele]), self.tree.tree.get_nonterminals())
                     print('Found %d partial matches for allele '%len(partial_matches), allele)
 
+    def annotate_fitness(self):
+        """Run the fitness prediction model and annotate the tree's nodes with fitness
+        values. Returns the resulting fitness model instance.
+        """
+        kwargs = {
+            "tree": self.tree.tree,
+            "frequencies": self.tree_frequencies,
+            "time_interval": self.info["time_interval"],
+            "pivot_spacing": self.config["pivot_spacing"]
+        }
+
+        if "predictors" in self.config:
+            kwargs["predictor_input"] = self.config["predictors"]
+
+        if "titers" in self.config:
+            if "epitope_mask" in self.config["titers"]:
```

Comment:
Feels semantically strange to have `epitope_mask` under `titers`. These are separate things to me. (Expanding on this in the main PR thread)